### PR TITLE
tidied up 3 spots of Array handling per #7

### PR DESCRIPTION
**Pull request by ndushay**
also:
- set rubocop line length to 120
- update email to triannon-commits list

@darrenleeweber 

connected to #7

### CODE REVIEW

**Comment by dazza-codes** [comment]
Waiting on travis...

File: .rubocop.yml [hunk 1]
```diff
@@ -31,8 +31,8 @@ Metrics/ClassLength:
   CountComments: false  # count full line comments?
   Max: 150
 
-#Metrics/LineLength:
-#  Max: 180
+Metrics/LineLength:
+  Max: 120
```
Comment (dazza-codes) [inline .rubocop.yml:35]: OK, sneak it into this PR.

File: .travis.yml [hunk 2]
```diff
@@ -10,5 +10,4 @@ env:
     - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
 notifications:
   email:
-    - ndushay@stanford.edu
-    - darren.weber@stanford.edu
+    - triannon-commits@lists.stanford.edu
```
Comment (dazza-codes) [inline .travis.yml:13]: OK, sneak it into this PR.

File: .rubocop.yml (hunk 1 above)
Comment (ndushay) [inline .rubocop.yml:35]: :+1:

File: .travis.yml (hunk 2 above)
Comment (ndushay) [inline .travis.yml:13]: also :+1:

File: lib/oa/graph.rb [hunk 3]
```diff
@@ -78,30 +78,17 @@ def id_as_url
...
-      solution = @graph.query q
-      if solution && solution.size > 0
-        solution.each { |res|
-          motivations << res.motivated_by.to_s
-        }
-      end
-      # TODO:  raise exception if none? (validation)
-      motivations
+      # if the motivation object gives junk with .to_s, then the anno is illegal anyway and we're not responsible
+      @graph.query(q).each.collect {|stmt| stmt.motivated_by.to_s}
```
Comment (dazza-codes) [inline lib/oa/graph.rb:84]: Yes.  Perhaps we need a new issue to focus on whether to coerce the result with `.to_s`.

File: lib/oa/graph.rb [hunk 4]
```diff
@@ -78,30 +78,17 @@ def id_as_url
...
     # @return [Array<String>] urls for the predicate, as an Array of Strings
     def predicate_urls(predicate)
-      urls = []
-      predicate_solns = @graph.query [nil, predicate, nil]
-      predicate_solns.each { |predicate_stmt|
-        predicate_obj = predicate_stmt.object
-        urls << predicate_obj.to_str.strip if predicate_obj.is_a?(RDF::URI)
-      }
-      urls
+      @graph.query([nil, predicate, nil]).each_object.collect {|o| o.to_str.strip if o.is_a?(RDF::URI) }.compact
```
Comment (dazza-codes) [inline lib/oa/graph.rb:91]: Again, using collect is simpler, that's great, although let's have an issue for the `o.to_str` etc.

File: oa-graph.gemspec [hunk 5]
```diff
@@ -6,7 +6,7 @@ Gem::Specification.new do |spec|
   spec.name          = 'oa-graph'
   spec.version       = OA::Graph::VERSION
   spec.authors       = ['Naomi Dushay']
-  spec.email         = ['ndushay@stanford.edu', 'darren.weber@stanford.edu']
+  spec.email         = ['triannon-commits@lists.stanford.edu']
```
Comment (dazza-codes) [inline oa-graph.gemspec:9]: OK, sneak it into this PR.

File: lib/oa/graph.rb (hunk 4 above)
Comment (ndushay) [inline lib/oa/graph.rb:91]: i hear you.  At least there's an "if" statement for it.  :smile:

**Comment by dazza-codes** [comment]
BTW, let's discuss using a `develop` branch for merging `feature` branches.  When we cut a release, the `develop` branch is merged into `master` and the version bumped.

**Comment by ndushay** [comment]
sure - let's discuss that f2f.

**Pull request merged**